pymeasure/instruments/__init__.py

```diff
@@ -24,9 +24,9 @@
 
 from ..errors import RangeError, RangeException
 from .instrument import Instrument
-from .mock import Mock
 from .resources import list_resources
 from .validators import discreteTruncate
+from .fakes import SwissArmyFake, FakeInstrument
```

Comment:
I would actually not even include them in the `instruments` namespace, as they are used for testing only.

Reply:
Good point. Changing this now.